src/lib/y2storage/disk.rb

```diff
@@ -0,0 +1,101 @@
+require "y2storage/storage_class_wrapper"
+require "y2storage/partitionable"
+require "y2storage/free_disk_space"
+
+module Y2Storage
+  # A physical disk device
+  #
+  # This is a wrapper for Storage::Disk
+  class Disk < Partitionable
+    include StorageClassWrapper
+    wrap_class Storage::Disk
+
+    storage_forward :rotational
+    alias_method :rotational?, :rotational
```

Comment:
Maybe. It sounds interesting. The Ruby hacker within me want to implement it, but I'm not sure about the benefits. For documentation purposes it's probably better to explicitly use `alias_method` and there is no need to add another level of magic just to save a few characters.